dojo/templates/dojo/view_test.html

```diff
@@ -551,6 +551,9 @@ <h4 class="has-filters"> Findings ({{findings.total_count}}) <small>{{ test.id|g
                             <option value="High">High</option>
                             <option value="Critical">Critical</option>
                         </select><br/><br/>
+                        <label><b>Planned Remediation Date</b><br/>
+                        <input id="id_bulk_planned_remediation_date" name="planned_remediation_date" type="date"
```

Comment:
The placeholder **TT.MM.JJJJ** is bold and black and doesn't look very nice. Can we have this in a normal font with a grey color?